azure/template.json

```diff
@@ -77,10 +84,45 @@
                     },
                     "storageKind": {
                         "value": "[parameters('storageKind')]"
+                    },
+                    "enableCors": {
+                        "value": "[parameters('enableCors')]"
                     }
                 }
             },
-            "dependsOn": []
+            "dependsOn": [
+            ]
+        },
+        {
+            "apiVersion": "2017-05-10",
+            "name": "[concat('storage-container', parameters('containerArray')[copyIndex()].name)]",
+            "type": "Microsoft.Resources/deployments",
+            "condition": "[greater(length(parameters('containerArray')), 0)]",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "https://raw.githubusercontent.com/SkillsFundingAgency/das-platform-building-blocks/master/templates/storage-container.json",
+                    "contentVersion": "1.0.0.0"
+                },
+                "parameters": {
+                    "storageAccountName": {
+                        "value": "[parameters('storageAccountName')]"
+                    },
+                    "containerName": {
+                        "value": "[parameters('containerArray')[copyIndex()].name]"
+                    },
+                    "publicAccess": {
+                        "value": "[parameters('containerArray')[copyIndex()].publicAccess]"
+                    }
```

Comment:
Could publicAccess be hardcoded to be None?

Reply:
No Because in this end it's Blob level as it's the source for the CDN